app/views/admin/content_assets/_clipboard.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class='copy-to-clipboard'>
-  <p><label for='clipboard_content'>Asset URL</label></p> 
+  <p><label class="govuk-label" for='clipboard_content'>Asset URL</label></p> 
   <p><input id='clipboard_content' name='clipboard_content' value='<%= clipboard_content %>' class='govuk-input' readonly></p>
-  <input id='clipboard_copier' type="button" onclick="" value="copy to clipboard">
-</div>
\ No newline at end of file
+  <input id='clipboard_copier' type="button" onclick="" value="Copy to clipboard">
```

Comment:
The two changes (adding the class to the label and updating the value) in this file look good, but I wonder if they are part of this bit of work, or have slipped in from some other work.